liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -299,6 +298,9 @@ protected void copyConfigFiles() throws Exception {
 
         // copy jvm.options to server directory if end-user explicitly set it
         File optionsFile = new File(serverDirectory, "jvm.options");
+        if (optionsFile.exists() && !configDirJvmOptionsFile.exists()) {
```

Comment:
The configDirJvmOptionsFile variable is not available here. It is defined inside an if statement above. You should just check to see if the jvmOptionsPath or bootStrapPropertiesPath variables are not null.